packages/cozy-scanner/src/DocumentCategory.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+import { Icon, Media, Bd, Img, Bold, IconStack } from 'cozy-ui/transpiled/react'
+
+import ActionMenu, {
+  ActionMenuItem,
+  ActionMenuHeader
+} from 'cozy-ui/transpiled/react/ActionMenu'
+import palette from 'cozy-ui/transpiled/react/palette'
+import GridItem from 'cozy-ui/transpiled/react/Labs/GridItem'
+
+import CategoryGridItem from './CategoryGridItem'
+
+import styles from './styles.styl'
+
+/**
+ *
+ * The goal of this component is to display a
+ * category / type of document and also manage
+ * its associated ActionMenu since a category has
+ * several associated items.
+ *
+ * If an item from a category is selected, we display
+ * its label and the label from the category
+ *
+ */
+class DocumentCategory extends Component {
+  state = {
+    isMenuDisplayed: false
+  }
+
+  toggleMenu() {
+    this.setState(prevState => {
+      return {
+        isMenuDisplayed: !prevState.isMenuDisplayed
+      }
+    })
+  }
+
+  onSelect = item => {
+    const { onSelect } = this.props
+    if (onSelect) onSelect(item)
+  }
+  render() {
+    const { isMenuDisplayed } = this.state
+    const { category, isSelected, selectedItem, items, t } = this.props
+    return (
+      <>
+        <GridItem onClick={() => this.toggleMenu()}>
+          <CategoryGridItem
```

Comment:
CategoryBox maybe ?

It was unclear to me at this point that GridItem was only dealing with the Layout where CategoryGridItem deals with the border and the selection.